storage/blob.go

```diff
@@ -473,44 +459,29 @@ func (b BlobStorageClient) ContainerExists(name string) (bool, error) {
 }
 
 // SetContainerPermissions sets up container permissions as per https://msdn.microsoft.com/en-us/library/azure/dd179391.aspx
-func (b BlobStorageClient) SetContainerPermissions(container string, containerPermissions ContainerPermissions) (err error) {
+func (b BlobStorageClient) SetContainerPermissions(container string, containerPermissions ContainerPermissions, timeout int, leaseID string) (err error) {
```

Comment:
[nit] I'd make `timeout` a `time.Duration` and make the call using something like this:
``` go
int(math.Ceil(timeout.Seconds()))
```